### ISSUE TITLE
Add wheel and windows builds on pypi

### ISSUE THREAD

**Issue by bendtherules** [duggan/pontoon#37]
[Pontoon on PyPi](https://pypi.python.org/pypi/pontoon) doesnt have wheel and other builds like .msi or .exe for windows. 

Although the later ones are not that important, wheels are the new standard for python packaging and so, we should definitely use that. As they are pre-compiled, they are also easier to install in some cases.

Appveyor has a way of building stuffs within it and making those packages available for download called artifacts - which is already setup and available for each job. For eg, here is one https://ci.appveyor.com/project/duggan/pontoon/build/1.0.24/job/6f0truodogexy7ul/artifacts . 

We can directly upload that during release to pypi or build it on dev machines using `python setup.py bdist_wheel bdist_wininst bdist_msi`. I am not sure if the windows builds are possible on linux machines. Also, we should build it using different major versions of Python.

**Comment by duggan** [comment]
I'm planning on having builds pushed to PyPI from Travis, which will cover the Linux builds; it [looks like AppVeyor provides similar functionality](http://kirbyfan64.github.io/posts/using-appveyor-to-distribute-python-wheels.html). Will do them both at the same time.

**Comment by bendtherules** [comment]
Cool, cant wait for the next release. :grinning:

**Comment by duggan** [comment]
I have these mostly working in a branch now, so next release will contain wheels and .exe builds!

I've had trouble getting the MSI uploaded to PyPI, but that can be tackled separately.

**Comment by bendtherules** [comment]
What is the trouble with MSI? Are you trying it at testpypi?

**Comment by bendtherules** [comment]
Also, I see `pontoon-0.2.0.win-amd64.exe` exists, but not the 32-bit version. Any trouble with that too?

**Comment by duggan** [comment]
MSI upload at testpypi is returning errors, yes.

I've only been building the 64 bit versions while testing, I expect 32 bit will work ok.

**Comment by duggan** [comment]
Logs for the MSI push failures are [here](https://ci.appveyor.com/project/duggan/pontoon/build/1.0.55/job/irayg2u4ma5tvok1) (python setup.py bdist_msi upload) and [here](https://ci.appveyor.com/project/duggan/pontoon/build/1.0.53/job/k58pd7078ar04jpo) (twine upload) if you want to have a look.

**Comment by bendtherules** [comment]
Yes, I am also getting the same error playing with a different package.

Seems like everyone has it and apparently, this is the solution https://github.com/schmir/xdistutils#the-bdist_msi_fixed-command . Install `xdistutils` and use `python setup.py bdist_msi_fixed`.

But I could not get it working. `python setup.py bdist_msi_fixed` gives me invalid command `bdist_msi_fixed`. You can give a try, too.

If things dont work after all, we should just keep it as a long-running bug and move on for now.

**Comment by duggan** [comment]
:+1:

**Comment by duggan** [comment]
Builds work, but the `configure` command still needs an overhaul before it's Windows compatible. Just dropping a well formatted config file would work, though needs to be documented at least.

**Comment by duggan** [comment]
There's a wheel for 0.2.1 on PyPI, though I still have to finish out work on the autobuilds for the Windows specific builds.
